.github/workflows/tests.yml

```diff
@@ -118,13 +118,13 @@ jobs:
           MAINNET_VALIDATOR_URL: ${{ secrets.MAINNET_VALIDATOR_URL }}
         run: anchor test ${{ matrix.test.scope }} ${{ matrix.test.testSuffix }}
       - name: Test | delete remote badges branch
-        if: ${{ github.ref == 'ref/head/main' && contains(matrix.test.scope, 'sdk') }}
+        if: ${{ github.ref == 'refs/heads/main' && contains(matrix.test.scope, 'sdk') }}
         run: git push origin --delete ${{ matrix.test.scope }}/ci-badges
       - name: Test | create badges branch
-        if: ${{ github.ref == 'ref/head/main' && contains(matrix.test.scope, 'sdk') }}
+        if: ${{ github.ref == 'refs/heads/main' && contains(matrix.test.scope, 'sdk') }}
         run: git checkout -b ${{ matrix.test.scope }}/ci-badges
       - name: Test | code coverage
-        if: ${{ github.ref == 'ref/head/main' && contains(matrix.test.scope, 'sdk') }}
+        if: ${{ github.ref == 'refs/heads/main' && contains(matrix.test.scope, 'sdk') }}
```

Comment:
The example i got `ref/head` was likely outdated.

Reply:
<img width="764" alt="image" src="https://github.com/seabed-labs/pre-authorized-debit/assets/93599851/938d9c22-9656-4ac2-8d8f-ac21d24ab365">

Reply:
https://docs.github.com/en/actions/learn-github-actions/contexts

Reply:
```
The fully-formed ref of the branch or tag that triggered the workflow run. For workflows triggered by push, this is the branch or tag ref that was pushed. For workflows triggered by pull_request, this is the pull request merge branch. For workflows triggered by release, this is the release tag created. For other triggers, this is the branch or tag ref that triggered the workflow run. This is only set if a branch or tag is available for the event type. The ref given is fully-formed, meaning that for branches the format is refs/heads/<branch_name>, for pull requests it is refs/pull/<pr_number>/merge, and for tags it is refs/tags/<tag_name>. For example, refs/heads/feature-branch-1.
```